### PR TITLE
solution

### PR DESCRIPTION
**Pull request by ivanzh13**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,50 @@
...
+            appetite=7,
+            is_hungry=is_hungry
+        )
+
+    @staticmethod
+    def bring_slippers() -> None:
+        print("The slippers delivered!")
+
+
+def feed_animals(animal_list: list) -> int:
```
Comment (ArsenPidhoretskyi) [inline app/main.py:49]: 1. rename animal_list to animals
2. add inner type to annotation. list[Animal]

File: app/main.py [hunk 2]
```diff
@@ -1 +1,50 @@
...
+    return sum([animal.feed() for animal in animal_list])
```
Comment (ArsenPidhoretskyi) [inline app/main.py:50]:
```suggestion
    return sum(animal.feed() for animal in animal_list)
```
Convert to list is redundant

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Changes requested

File: app/main.py (hunk 1 above)
Comment (ivanzh13) [reply app/main.py:49]: Please explain how to use: list[Animal] or list[Animal | Cat | Dog ]

Comment (ArsenPidhoretskyi) [reply app/main.py:49]: Better to use list[Animal] it already means that will work with all child classes

**Review by vasylhnatiuk — APPROVED** [review]

**Pull request closed without merging**